modules/sno-adding-worker-nodes-to-sno-clusters-manually.adoc

```diff
@@ -79,22 +79,56 @@ $ curl -L $ISO_URL -o rhcos-live.iso
 
 . Use the {op-system} ISO and the hosted `worker.ign` file to install the worker node:
 
-.. Boot the target host using the {op-system} ISO and your preferred method of installation.
+.. Boot the target host with the {op-system} ISO and your preferred method of installation.
 
-.. When the target host has booted from the {op-system} ISO, open a console on the target host, and run the following `coreos-installer` command to install the system:
+.. When the target host has booted from the {op-system} ISO, open a console on the target host.
+
+.. If your local network does not have DHCP enabled, you need to specify the static IP of the worker node before running the {op-system} installation. For example, run the following commands on the target host console:
```

Comment:
IMHO, instead of "you need to specify the static IP" 
"you need to set the static IP"  or "you need to configure the static IP"